src/devices/cpu/i386/i386.cpp

```diff
@@ -1728,8 +1845,12 @@ uint8_t i386_device::read8_debug(uint32_t ea, uint8_t *data)
 {
 	uint32_t address = ea;
 
-	if(!i386_translate_address(TRANSLATE_DEBUG_MASK,&address,nullptr))
-		return 0;
+	if (!(m_cr[4] & 0x20))
+	{
+		if (!i386_translate_address_normal(TRANSLATE_DEBUG_MASK,&address,nullptr))
+			return 0;
+	}
+	else if (!i386_translate_address_pae(TRANSLATE_DEBUG_MASK,&address,nullptr))
 
 	address &= m_a20_mask;
 	*data = m_program->read_byte(address);
```

Comment:
I don’t know if it’s intentional, but having the controlled statement of the `else if` at the same indentation level and separated from it by a blank line is like intentionally making the code difficult to understand.